Solutions/Tanium/Analytic Rules/TaniumThreatResponseAlerts.yaml

```diff
@@ -0,0 +1,34 @@
+id: dd9aa0ff-7ac1-4448-879c-e1a18d5890b4
+name: Tanium Threat Response Alerts
+description: 'Alerts from Tanium Thread Response (THR) that can be acted upon by Tanium Sentinel Playbook'
```

Comment:
Do we have a Typo here: Thread vs Threat

Reply:
Thank you, I fixed this in the most recent commit here: 
https://github.com/Azure/Azure-Sentinel/pull/5307/commits/f581d295ef400200dc54ec77e8a279ba05bf8170